bigtop-packages/src/deb/flink/control

```diff
@@ -27,3 +27,13 @@ Depends: bigtop-utils (>= 0.7), adduser
 Description: Flink’s core is a streaming dataflow engine that provides data
              distribution, communication, and fault tolerance for distributed
              computations over data streams.
+
+Package: flink-jobmanager
+Architecture: all
+Depends: flink
```

Comment:
you need 
    Depends: flink (= ${source:Version})
here instead to make sure that there's 1-1 correspondence version-wise